wpilibj/src/athena/java/edu/wpi/first/wpilibj/DoubleSolenoid.java

```diff
@@ -34,15 +32,13 @@
     kReverse
   }
 
-  private int m_forwardChannel; // /< The forward channel on the module to control.
-  private int m_reverseChannel; // /< The reverse channel on the module to control.
-  private byte m_forwardMask; // /< The mask for the forward channel.
-  private byte m_reverseMask; // /< The mask for the reverse channel.
+  private byte m_forwardMask; // The mask for the forward channel.
+  private byte m_reverseMask; // The mask for the reverse channel.
```

Comment:
I notice you're removing the doxygen style line comments (although they're incorrectly formatted, as they should be `///<` without an intervening space).  Should we instead be fixing them by removing the incorrect space?